great_expectations_provider/operators/great_expectations.py

```diff
@@ -251,6 +252,18 @@ def make_connection_string(self) -> str:
                 odbc_connector = "mssql+pyodbc"
             uri_string = f"{odbc_connector}://{self.conn.login}:{self.conn.password}@{self.conn.host}:{self.conn.port}/{self.schema}"  # noqa
         elif conn_type == "snowflake":
+            try:
+                return self.build_snowflake_connection_config_from_hook()
+            except ImportError:
+                self.log.warning(
+                    (
+                        "Snowflake provider package could not be imported, "
+                        "attempt to build connection uri from %s "
```

Comment:
nit: attempt --> attempting

I think the "Key-based auth is not supported" should also come with a note of how to make it supported.

Reply:
I reworded the message.